misc/wallaroo-up.sh

```diff
@@ -50,8 +50,8 @@ W0.5.4=0.25.0
 W0.5.3=0.24.4
 W0.5.2=0.24.4
 Wuse-ponyc-0.25=0.25.0
-Wponyc-0.27.0=0.28.0
-Wlatest=0.28.0
+Wponyc-0.27.0=0.29.0
```

Comment:
the checksum for this needs to be updated as well. the build is currently failing.

Reply:
The following should be run to update the checksum:

```
  # update checksum in wallaroo-up.sh
  WALLAROO_UP_CHECKSUM_COMMAND=$(grep -Po '(?<=^CALCULATED_MD5="\$\().*(?=\)")' misc/wallaroo-up.sh | sed 's@\$0@misc/wallaroo-up.sh@')
  sed -i "s@^MD5=.*@MD5=\"$(eval $WALLAROO_UP_CHECKSUM_COMMAND)\"@" misc/wallaroo-up.sh
```

That should work, if it doesn't ping me

Reply:
also, `ponyc-0.27.0` should be updated to `update-0.29` in order for it to work with your branch